packages/ui-library/src/components/checkbox/index.stories.ts

```diff
@@ -166,7 +165,7 @@ export default {
       },
     },
     //Technical attributes
-    checkInputId: {
+    checkboxId: {
       name: 'checkInputId',
```

Comment:
I think you need to remove 'checkInputId' as a name for this argument. I would also remove the 'name' property from each argument unless the display name doesn't match the provided argument. For example, with
`    errorMessageIcon: {
      name: 'errorMessageIcon',
}`
the `name: 'errorMessageIcon',` isn't needed since it will take `errorMessageIcon` as the name by default

Reply:
done